ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -85,6 +96,53 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28
+  capabilities:
+  - intranet
+  cron: 36 16 8 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      ENABLE_OTA_TEST: OCP-25473
```

Comment:
This profile is also needed in 4.18-4.19 upgrade?

Reply:
for any upgrade path requires 4.20 are still not supported because there is no stable 4.10 payload.

Reply:
done